latest_major_version.go

```diff
@@ -0,0 +1,117 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016 Datadog, Inc.
+
+
+package main
```

Comment:
The linter job complains about this file not being `gofmt`'ed.

I can help setting the editor (format-on-save, etc.) when you have some time.